### ISSUE TITLE
error "Version must be a string"

### ISSUE THREAD

**Issue by seagle0128** [magit/ghub#91]
GNU Emacs 27.0.50
macOS Mojave
ghub-20190523.1139

```
Debugger entered--Lisp error: (error "Version must be a string")
  signal(error ("Version must be a string"))
  error("Version must be a string")
  version-to-list(27)
  version<=(27 "26.2")
  (and (>= libgnutls-version 30603) (version<= emacs-major-version "26.2") 'force)
  (defvar ghub-use-workaround-for-emacs-bug (and (>= libgnutls-version 30603) (version<= emacs-major-version "26.2") 'force) "Whether to use a kludge that hopefully works aroun...")
  eval-buffer(#<buffer  *load*-824322> nil "/Users/vincent/.emacs.d/elpa/ghub-20190523.1139/gh..." nil t)  ; Reading at buffer position 20186
  load-with-code-conversion("/Users/vincent/.emacs.d/elpa/ghub-20190523.1139/gh..." "/Users/vincent/.emacs.d/elpa/ghub-20190523.1139/gh..." nil t)
  require(ghub)
  eval-buffer(#<buffer  *load*-332998> nil "/Users/vincent/.emacs.d/elpa/forge-20190508.501/fo..." nil t)  ; Reading at buffer position 855
  load-with-code-conversion("/Users/vincent/.emacs.d/elpa/forge-20190508.501/fo..." "/Users/vincent/.emacs.d/elpa/forge-20190508.501/fo..." nil t)
  require(forge-github)
  eval-buffer(#<buffer  *load*> nil "/Users/vincent/.emacs.d/elpa/forge-20190508.501/fo..." nil t)  ; Reading at buffer position 1527
  load-with-code-conversion("/Users/vincent/.emacs.d/elpa/forge-20190508.501/fo..." "/Users/vincent/.emacs.d/elpa/forge-20190508.501/fo..." nil t)
  require(forge nil nil)
  (closure (t) nil (require 'forge nil nil))()
  eval-after-load(magit (closure (t) nil (require 'forge nil nil)))
  (progn (eval-after-load 'magit #'(lambda nil (require 'forge nil nil))))
  (if (executable-find "cc") (progn (eval-after-load 'magit #'(lambda nil (require 'forge nil nil)))))
  (progn (use-package-ensure-elpa 'forge '(t) '(:demand t)) (if (executable-find "cc") (progn (eval-after-load 'magit #'(lambda nil (require 'forge nil nil))))))
  (progn (progn (use-package-ensure-elpa 'forge '(t) '(:demand t)) (if (executable-find "cc") (progn (eval-after-load 'magit #'(lambda nil (require ... nil nil)))))))
  eval((progn (progn (use-package-ensure-elpa 'forge '(t) '(:demand t)) (if (executable-find "cc") (progn (eval-after-load 'magit #'(lambda nil ...)))))) t)
  elisp--eval-last-sexp(nil)
  eval-last-sexp(nil)
  funcall-interactively(eval-last-sexp nil)
  call-interactively(eval-last-sexp nil nil)
  #f(compiled-function (cmd &optional record-flag keys special) "Execute CMD as an editor command.\nCMD must be a symbol that satisfies the `commandp' predicate.\nOptional second arg RECORD-FLAG non-nil\nmeans unconditionally put this command in the variable `command-history'.\nOtherwise, that is done only if an arg is read using the minibuffer.\nThe argument KEYS specifies the value to use instead of (this-command-keys)\nwhen reading the arguments; if it is nil, (this-command-keys) is used.\nThe argument SPECIAL, if non-nil, means that this command is executing\na special event, so ignore the prefix argument and don't clear it." #<bytecode 0x430c45a7>)(eval-last-sexp nil nil nil)
  ad-Advice-command-execute(#f(compiled-function (cmd &optional record-flag keys special) "Execute CMD as an editor command.\nCMD must be a symbol that satisfies the `commandp' predicate.\nOptional second arg RECORD-FLAG non-nil\nmeans unconditionally put this command in the variable `command-history'.\nOtherwise, that is done only if an arg is read using the minibuffer.\nThe argument KEYS specifies the value to use instead of (this-command-keys)\nwhen reading the arguments; if it is nil, (this-command-keys) is used.\nThe argument SPECIAL, if non-nil, means that this command is executing\na special event, so ignore the prefix argument and don't clear it." #<bytecode 0x430c45a7>) eval-last-sexp)
  apply(ad-Advice-command-execute #f(compiled-function (cmd &optional record-flag keys special) "Execute CMD as an editor command.\nCMD must be a symbol that satisfies the `commandp' predicate.\nOptional second arg RECORD-FLAG non-nil\nmeans unconditionally put this command in the variable `command-history'.\nOtherwise, that is done only if an arg is read using the minibuffer.\nThe argument KEYS specifies the value to use instead of (this-command-keys)\nwhen reading the arguments; if it is nil, (this-command-keys) is used.\nThe argument SPECIAL, if non-nil, means that this command is executing\na special event, so ignore the prefix argument and don't clear it." #<bytecode 0x430c45a7>) eval-last-sexp)
  command-execute(eval-last-sexp)
```

**Comment by npostavs** [comment]
Oops, I had tested in an Emacs with an older `libgnutls-version` so I never actually evaluated the `(version<= ...)` part. Should be fixed in 0ca5979eb415022ce85d8190fbc9495cffcf9962.

**Comment by seagle0128** [comment]
Thank you!
